ir/memory.cpp

```diff
@@ -304,10 +312,27 @@ void Memory::memset(const expr &p, const StateValue &val, const expr &bytes,
 
 void Memory::memcpy(const expr &d, const expr &s, const expr &bytes,
                     unsigned align_dst, unsigned align_src) {
+  // TODO: Add ub condition - bytes == 0
```

Comment:
this requites some API changes. Either address it in this PR or in a subsequent patch.

Reply:
I'm not sure that when 'bytes' is zero and 'src (or tgt) pointer' is poison, the code has ub or not.

I am going to address similar cases in a subsequent patch.